kitchen-terraform.gemspec

```diff
@@ -21,7 +32,7 @@ require "rubygems"
   specification.homepage = "https://newcontext-oss.github.io/kitchen-terraform/"
   specification.license = "Apache-2.0"
   specification.add_development_dependency "bundler", "~> 2.0"
-  specification.add_development_dependency "guard-bundler", "~> 2.1"
+  specification.add_development_dependency "delegate", "~> 0.1.0"
```

Comment:
What is the motivation for adding the delegate gem?

Reply:
lib/kitchen/terraform/debug_logger.rb **require**s it.

Reply:
Interesting. I didn't realize that gem had been broken out of the Ruby stdlib.

Reply:
Can we mark it as a runtime dependency?

Reply:
I assumed, since it was required by debug_logger, that would just be a development need. Is that wrong?

Reply:
That is indeed not correct. `DebugLogger` is used to log messages at the debug level when reading Terraform outputs during the converge action: https://github.com/newcontext-oss/kitchen-terraform/blob/0b1dc3ff61397f49767c6131710591c7b63e2176/lib/kitchen/terraform/provisioner/converge.rb#L152

The motivation was that users did not want sensitive outputs to appear in their Kitchen logs, and it was a compromise to hide the outputs unless debug logging is enabled.

Reply:
Ok, I added it to runtime with the latest push.